cf/commands/service/create_service.go

```diff
@@ -60,6 +66,13 @@ func (cmd CreateService) Run(c *cli.Context) {
 	serviceName := c.Args()[0]
 	planName := c.Args()[1]
 	serviceInstanceName := c.Args()[2]
+	params := c.String("c")
+
+	paramsMap := make(map[string]interface{})
+	err := json.Unmarshal([]byte(params), &paramsMap)
+	if err != nil && params != "" {
+		cmd.ui.Failed("Invalid JSON provided in parameters argument")
```

Comment:
This string (if it might end up shown to end user) should be in `T("...")`

Here is the French translation, if you do add it:

```
"JSON non-valide fourni dans l'argumentation des paramètres"
```